neo/SDK/TX/TransactionContext.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Neo.Network.P2P.Payloads;
+using Neo.SmartContract;
+using System.Collections.Generic;
+
+namespace Neo.SDK.TX
```

Comment:
Spell out Transaction

Reply:
Transaction namespace will be conflict with Transaction type😥
![image](https://user-images.githubusercontent.com/4393234/63079498-140b3800-bf71-11e9-9d61-88cc2c46b6d7.png)

Reply:
Then just remove the namespace. The types already have "Transaction" in the name, so there's not much need for the namespace here.